### ISSUE TITLE
Create mockup of a more usable and aesthetically pleasing notifications view

### ISSUE THREAD

**Issue by mdconatser** [umgc/fall2022#237]
The customer has a strong desire for a better overall layout of the application. Mockups are the most effective way to demonstrate our ideas. Please include mockups as an attachment to this issue and the team can provide feedback, after which the SRS can be updated with new mockups.

Ensure it is clear that both text content within the email can support a notification, as well as the OCR results of a mail image.

**Comment by cthorn2** [comment]
Here is a mockup of the notifications view with add/remove notifications and go to message.

![Image](https://user-images.githubusercontent.com/111912838/189466605-eba2cea4-512c-4bc4-a7d0-2a9bc2dc5450.png)

**Comment by mdconatser** [comment]
- if there are matches on multiple days for "BGE" does it show multiple lines here? if so, its a little unclear if the Remove button clears the notification or removes the keyword notification. 
- May benefit from a "Manage Notifications" view instead for deleting so it's easier to see what you have actually set without duplicates. 
- probably would put "Go to Message" before "Remove" (or Clear)

**Comment by cthorn2** [comment]
Thanks for the feedback.  Here are some updated wireframes.

From the notifications view, the user has the ability to manage notification (those I believe should live in the settings, but open to suggestions).  The Go to Message, has been moved to the front of the notification.  One question, would it be better for accessibility to order this after the keywords column?  Then I have placed the clear button to clear just that notification.  There is an add button to add a keyword notification.

The manage notification screens, show that each line is provided with an edit and clear button.  The edit replaces the existing line and fills in with a two button form.  The clear removes the line immediately..



![Image](https://user-images.githubusercontent.com/111912838/189492603-227b2422-12e4-4723-a762-649b8709ac44.png)
